### ISSUE TITLE
Checkbox markers are not visible with bright theming color

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#14156]
Theme color: `#F1F1F1`

> ![bildschirmfoto von 2019-02-12 10-15-48](https://user-images.githubusercontent.com/213943/52624633-747cc800-2eaf-11e9-8d5a-e211da0adba1.png)

It gets even worse on dark theme
> ![bildschirmfoto von 2019-02-12 10-15-57](https://user-images.githubusercontent.com/213943/52624636-75adf500-2eaf-11e9-9383-e37d6e7d3301.png)

I guess the checkmark should be the color that is used for the header icons too instead?
@nextcloud/designers

**Comment by jancborchardt** [comment]
> I guess the checkmark should be the color that is used for the header icons too instead?

Yes, that seems best for uniformity, good proposal. :+1: 

cc @juliushaertl as you worked on the checkboxes part, correct?
